src/south/SQLDbToFile/SQLDbToFile.class.js

```diff
@@ -228,9 +258,11 @@ class SQLDbToFile extends ProtocolHandler {
     try {
       const pool = await new mssql.ConnectionPool(config).connect()
       let result
-      if (this.containsLastCompletedDate) {
+      if (this.nrOfReplacements) {
         result = await pool.request()
-          .input('LastCompletedDate', mssql.DateTimeOffset, lastCompletedAt)
+          .input('StartTime', mssql.DateTimeOffset, startTime)
+          .input('EndTime', mssql.DateTimeOffset, endTime)
+          .input('MaxReturnValues', mssql.Numeric, this.maxReturnValues)
```

Comment:
We noticed that DateTimeOffset does not support date operation in MSSQL queries. Is there any reason why to use that instead of DateTime ?